x-pack/legacy/plugins/lens/server/routes/index_stats.ts

```diff
@@ -95,6 +96,116 @@ export async function initStatsRoute(setup: CoreSetup) {
       }
     }
   );
+
+  router.post(
```

Comment:
It's probably best to give each endpoint its own file, so the code is focused and contained, otherwise it starts to blur together when looking through the one file.